base/multidimensional.jl

```diff
@@ -826,7 +826,7 @@ julia> y
 """
 copyto!(dest, src)
 
-function copyto!(dest::AbstractArray{T,N}, src::AbstractArray{T,N}) where {T,N}
+function copyto!(dest::AbstractArray{T1,N}, src::AbstractArray{T2,N}) where {T1,T2,N}
```

Comment:
Just use `AbstractArray{<:Any,N}` instead then since `T1` and `T2` are never used?

Reply:
Yeah, wish there was a shorter way of doing this thgouh. `AbstractArray{_,N}`is a syntax error.

Reply:
But does it mean `T->AbstractArray{T,N}`?

Reply:
:scream:

Reply:
Yeah, this is why you don't want to jump into new syntax too quickly...